packages/medusa-react/src/hooks/admin/sales-channels/queries.ts

```diff
@@ -13,6 +13,11 @@ export const adminSalesChannelsKeys = queryKeysFactory(
 
 type SalesChannelsQueryKeys = typeof adminSalesChannelsKeys
 
+/**
+ * @experimental This feature is still under development and can't be used in production
```

Comment:
todo: Please use exact same format as seen in [this PR](https://github.com/medusajs/medusa/pull/1811/files)

Note, we'll add it to the Code Convention doc 👍